experimental/logging/log_listener_example_test.go

```diff
@@ -55,3 +56,43 @@ func Example_newLoggingListenerFactory() {
 	//	<== ESUCCESS
 	//<-- ()
 }
+
+// This example shows how to see all function calls, including between host
+// functions.
+func Example_newLoggingListenerFactory() {
+	// Set context to one that has an experimental listener
+	ctx := context.WithValue(context.Background(), experimental.FunctionListenerFactoryKey{}, logging.NewLoggingListenerFactory(os.Stdout))
+
+	r := wazero.NewRuntimeWithConfig(ctx, wazero.NewRuntimeConfigInterpreter())
+	defer r.Close(ctx) // This closes everything this Runtime created.
+
+	wasi_snapshot_preview1.MustInstantiate(ctx, r)
+
+	// Compile the WebAssembly module using the default configuration.
+	code, err := r.CompileModule(ctx, listenerWasm)
+	if err != nil {
+		log.Panicln(err)
+	}
+
+	mod, err := r.InstantiateModule(ctx, code, wazero.NewModuleConfig().WithStdout(os.Stdout))
+	if err != nil {
+		log.Panicln(err)
+	}
+
+	_, err = mod.ExportedFunction("rand").Call(ctx, 4)
+	if err != nil {
+		log.Panicln(err)
+	}
+
+	// We should see the same function called twice: directly and indirectly.
+
+	// Output:
+	//--> listener.rand(len=4)
+	//	--> listener.wasi_rand(len=4)
+	//		==> wasi_snapshot_preview1.random_get(buf=4,buf_len=4)
+	//		<== ESUCCESS
+	//		==> wasi_snapshot_preview1.random_get(buf=8,buf_len=4)
+	//		<== ESUCCESS
+	//	<-- ()
+	//<-- ()
```

Comment:
seems all logged

Reply:
yeah this is distracting but intentional. I wanted an integration test to show the old logger will show all calls still.